Src/AutoFixtureUnitTest/Kernel/CriterionTests.cs

```diff
@@ -94,9 +94,9 @@ public void SutEqualsIdenticalValue()
         [InlineData(typeof(Version))]
         public void SutDoesNotEqualAnyObject(object other)
         {
-            var sut = new Criterion<PlatformID>(
-                PlatformID.Unix,
-                new DelegatingEqualityComparer<PlatformID>());
+            var sut = new Criterion<ConsoleColor>(
+                ConsoleColor.Cyan,
+                new DelegatingEqualityComparer<ConsoleColor>());
             var actual = sut.Equals(other);
```

Comment:
Yep. The `PlatformID` and a few other types are missing, so I tried to find a replacement for them. In this particular example I replaced one enum with other one (while that still doesn't matter for the test).

That was the most complex part of the migration - to ensure that replacements are relevant.